docs/terminal/sydney.md

```diff
@@ -14,28 +14,31 @@
 | Sydney Departures South†   |SDS| Sydney Departures | 129.700          | SY-S_DEP         |
 | Sydney Director West† |SFW| Sydney Director   | 126.100          | SY-D_APP                               |
 | Sydney Director East† |SFE| Sydney Director   | 125.300          | SY-DE_APP                               |
+| Sydney Radar† |SRI| Sydney Centre  | 125.800          | SY-D_DEP                               |
```

Comment:
Be good to discuss whether we can make the login identifier linking it to Sydney Centre?

Reply:
I thought we discussed in AIS channel there were too many implications that could make it hard

Reply:
freq 124.55 is already in VATSYS...

Logon identifier: SY-C_DEP